CoffeeMachine.java

```diff
@@ -0,0 +1,90 @@
+package Lesson_12;
+
+import java.util.Scanner;
+
+public class CoffeeMachine {
+    private String esspresso = "Esspresso";
+    private String macchiato = "Macchiato";
+    private String cappuccino = "Cappuccino";
+    private String milk = "Milk";
+    private String hotChocolate = "Hot Chocolate";
+    private String drinkChoice;
+    private int answer;
+    private char choice;
+    private String[] drinkList = {"Espresso", "Macchiato", "Cappuccino"};
+
+    public CoffeeMachine(){
+        this.esspresso = esspresso;
+        this.macchiato = macchiato;
+        this.cappuccino = cappuccino;
+        this.milk = milk;
+        this.hotChocolate = hotChocolate;
+        this.drinkChoice = drinkChoice;
+        this.answer = answer;
+        this.drinkList=drinkList;
+        this.choice=choice;
+    }
+
+    public void getDrink(){
+        System.out.println("Choose your drink");
+        int n = 0;
+        for (String menu:this.drinkList){
+            System.out.println(++n+ " | " + menu);
+        }
+        makeChoice();
+    }
+
+    private void makeChoice(){
+        Scanner reader = new Scanner(System.in);
+        this.answer = reader.nextInt();
+
+        if (this.answer==1){
```

Comment:
What if answer is some other value than 1, 2, 3 ? 
It would be good to add else at the end which captures any other user input and just prints something like:
"Incorrect option chosen. Please choose 1, 2 or 3"